slackbridge/messages.py

```diff
@@ -12,12 +15,20 @@
     'channel_leave',
 )
 
+# Extensions that we will allow to be mirrored to IRC
+ALLOWED_FILE_EXTS = (
+    'jpg',
+    'png',
+    'gif',
+)
```

Comment:
Imgur does seem to allow [more file types](https://help.imgur.com/hc/en-us/articles/115000083326-What-files-can-I-upload-What-is-the-size-limit-), why is this limit here? That being said, it can't handle arbitrary files, so this might be one good reason to use Fluffy, in that it can accept any type of file, and has a pretty simple API ([as shown by one of the CLI scripts](https://github.com/chriskuehl/fluffy/blob/31e559867c295c9858893a2ec8d19cf2693f93c0/cli/fluffy_cli/main.py#L62)) that doesn't need authentication.